recipes/terminus/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "v0.1.0" %}
+
+package:
+  name: terminus
+  version: {{version}}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/COMBINE-lab/terminus/archive/{{ version }}.tar.gz
+  sha256: 7be85800e5977f04dcf72e5205deb369a2033a068e0adcec46e3a61d7edda96c
+
+requirements:
+  build:
+    - rust >=1.40
+    - clangdev
+    - {{ compiler('cxx') }}
+    - pkg-config
+  host:
+    - clangdev
+    - gsl
+    - libopenblas
```

Comment:
Can you please follow the blas guidelines from conda-forge here: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=blas#blas

Reply:
Apologies for my ignorance, this is my first PR with rust package. Following the guideline, are you suggesting to add all the "blas" packages as specified there?